core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -117,6 +124,13 @@ public class ExchangeFileSystem extends MailFileSystemBase<EmailMessage,Attachme
 	private @Getter String proxyAuthAlias = null;
 	private @Getter String proxyDomain = null;
 
+	private final String AUTHORITY = "https://login.microsoftonline.com/";
+	private final String SCOPE = "https://outlook.office365.com/.default";
+	private @Getter @Setter String clientId = null;
+	private @Getter @Setter String clientSecret = null;
+	private @Getter @Setter String tenant = null;
```

Comment:
To let this work effectively, these attributes must be made available on ExchangeMaiListener and ExchangeMailSender too.
Please also add IbisDoc or JavaDoc to the setters, describing
- what is necessary to configure
- that they override username and password